test/metabase/api/downloads_exports_test.clj

```diff
@@ -198,21 +231,45 @@
                      :model/Card {pivot-card-id :id}
                      {:display                :pivot
                       :visualization_settings {:pivot_table.column_split
-                                               {:rows    [[:field "CATC" {:base-type :type/Text}]
-                                                          [:field "CATD" {:base-type :type/Text}]]
-                                                :columns [[:field "CATA" {:base-type :type/Text}]
-                                                          [:field "CATB" {:base-type :type/Text}]]
+                                               {:rows    [[:field "C" {:base-type :type/Text}]
+                                                          [:field "D" {:base-type :type/Text}]]
+                                                :columns [[:field "A" {:base-type :type/Text}]
+                                                          [:field "B" {:base-type :type/Text}]]
                                                 :values  [[:aggregation 0]]}}
                       :dataset_query          {:database (mt/id)
                                                :type     :query
                                                :query
                                                {:aggregation  [[:sum [:field "MEASURE" {:base-type :type/Integer}]]]
                                                 :breakout
-                                                [[:field "CATA" {:base-type :type/Text}]
-                                                 [:field "CATB" {:base-type :type/Text}]
-                                                 [:field "CATC" {:base-type :type/Text}]
-                                                 [:field "CATD" {:base-type :type/Text}]]
+                                                [[:field "A" {:base-type :type/Text}]
+                                                 [:field "B" {:base-type :type/Text}]
+                                                 [:field "C" {:base-type :type/Text}]
+                                                 [:field "D" {:base-type :type/Text}]]
                                                 :source-table (format "card__%s" pivot-data-card-id)}}}]
         (let [result (mt/user-http-request :crowberto :post 200 (format "card/%d/query/json?format_rows=false" pivot-card-id))]
-            (take 20 result)))))
+          (take 20 result)))))
+
+  (defn multi-measure-pivot
```

Comment:
instead of a big `comment` form, could we make this a test?

Reply:
This function is equivalent to the `multi-measure-pivot-tables-headers-test` I added above. I forgot to remove this from the comment form, oops

Reply:
I've removed the comment form altogether. The functions that were in there are what I used to build the tests in the first place, so the namespace already contains the tests.